javascript-light-bulb/main.js

```diff
@@ -0,0 +1,12 @@
+var $button = document.querySelector('button');
+var $body = document.querySelector('body');
+
+$button.addEventListener('click', function (e) {
+  if ($body.className === 'dark') {
```

Comment:
Rather than making this functionality dependent on the current state of the output, you should have a variable here in the JavaScript to tell you if the light is currently on or off. What if the className got changed to an unexpected value by a future addition to the code? Everything would break 💀